internal/handler/insightsParserFilter.go

```diff
@@ -92,6 +93,7 @@ func processFactsFromSBOM(logger *slog.Logger, facts *[]cdx.Component, environme
 			Description: f.PackageURL,
 			KeyFact:     false,
 			Type:        FactTypeText,
+			Service:     strings.TrimPrefix(source, "insights:sbom:"),
```

Comment:
let's get this in ASAP (https://github.com/DependencyTrack/client-go/pull/40), and you make corresponding changes here to support it - namely, add the service field to the insights message, and explicitly use it right here (if you want to use the TrimPrefix as a fallback, feel free, but I don't think we need it)